msm/src/prover.rs

```diff
@@ -76,13 +251,24 @@ where
         };
         (evals(&zeta), evals(&zeta_omega))
     };
+    // TODO: Parallelize
+    let (mvlookup_zeta_evaluations, mvlookup_zeta_omega_evaluations) = {
+        let evals = |point| {
+            let comm = |poly: &DensePolynomial<G::ScalarField>| poly.evaluate(point);
```

Comment:
Why is it called `comm` and not smth like `eval`? It's not a commitment, right? IIRC it's scalar element.

Reply:
Oops. Copy paste. Fixed.